blockchainetl/jobs/exporters/kafka_exporter.py

```diff
@@ -10,17 +10,24 @@
 class KafkaItemExporter:
 
     def __init__(self, output, item_type_to_topic_mapping, converters=()):
-        self.item_type_to_topic_mapping = item_type_to_topic_mapping
+        self.item_type_to_topπic_mapping = item_type_to_topic_mapping
```

Comment:
`π` is added mistakenly to item_type_to_topπic_mapping

Reply:
Oh, great attention to detail. Thanks for spotting that!